### PR TITLE
Update the SpacesDashboard layout as per IA requirement

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Resolves
- https://github.com/sparkletown/internal-sparkle-issues/issues/1459

**Before**:

sparkle-spaces-dash-05

**After**:

sparkle-spaces-dash-06

sparkle-spaces-dash-04
sparkle-spaces-dash-03

### CODE REVIEW

**Comment by colinhowe** [comment]
There are some headers missing. They'e created as part of https://github.com/sparkletown/internal-sparkle-issues/issues/1437

File: src/components/atoms/SortDropDown/SortDropDown.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import "scss/constants";
+
+.SortDropDown {
+  // NOTE: this dropdown mimics the appearance of the ButtonNG default style
+  // NOTE: it is needed to mitigate the effects of the global SCSS file and Bootstrap
```
Comment (colinhowe) [inline src/components/atoms/SortDropDown/SortDropDown.scss:5]: :sob:

File: src/components/atoms/SortDropDown/SortDropDown.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  Dropdown as ReactBootstrapDropdown,
+  DropdownButton,
+} from "react-bootstrap";
+
+import { SortingOptions } from "utils/venue";
+
+import "./SortDropDown.scss";
+
+interface SpaceSortDropDownProps {
+  onClick?: (value: SortingOptions) => void;
```
Comment (colinhowe) [inline src/components/atoms/SortDropDown/SortDropDown.tsx:12]: Should this be optional?

I can't see any point in using this dropdown without a callback provided.

**Comment by goran-peoski-work** [comment]
> There are some headers missing. They'e created as part of [sparkletown/internal-sparkle-issues#1437](https://github.com/sparkletown/internal-sparkle-issues/issues/1437)

Different issue, different PR.

This PR is resolving https://github.com/sparkletown/internal-sparkle-issues/issues/1459

File: src/components/atoms/SortDropDown/SortDropDown.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+export const SortDropDown: React.FC<SpaceSortDropDownProps> = ({ onClick }) => (
```
Comment (colinhowe) [inline src/components/atoms/SortDropDown/SortDropDown.tsx:15]: I think this component is misnamed (or too specific in it's parameters).

As is, it's more of a `SpaceSortDropDown`. I think either:
 * rename it to `SpaceSortDropDown`
 * or, make it accept a more generic "sorting options" parameter

I don't have a strong opinion on which.

**Review by colinhowe — CHANGES_REQUESTED** [review]
Minor changes required. Once they're done go ahead and merge.

File: src/components/atoms/SortDropDown/SortDropDown.tsx (hunk 2 above)
Comment (goran-peoski-work) [reply src/components/atoms/SortDropDown/SortDropDown.tsx:12]: I was thinking about this, then I went to fix something else, I forgot I guess

File: src/components/atoms/SortDropDown/SortDropDown.tsx (hunk 3 above)
Comment (goran-peoski-work) [reply src/components/atoms/SortDropDown/SortDropDown.tsx:15]: Heh, that was the first name I gave it `SpaceSortDropDown`.
Then I noticed it was using this `VenueSortingOptions` enum and I thought "OK, I will rename that to `SpaceSortingOptions`...
And I noticed it is basically a general sort: `A-Z`, `Z-A`, `Oldest First`, `Newest First`... could basically be used for anything.

Hence the removal of the `Space` and `Venue` from the names

File: src/components/atoms/SortDropDown/SortDropDown.tsx (hunk 2 above)
Comment (goran-peoski-work) [reply src/components/atoms/SortDropDown/SortDropDown.tsx]: removed the `?`  @ b7425bd8196ff5942d4c257358cf401de6df8750

File: src/components/atoms/SortDropDown/SortDropDown.tsx (hunk 3 above)
Comment (goran-peoski-work) [reply src/components/atoms/SortDropDown/SortDropDown.tsx]: Removed the `Space` from `SpaceSortDropDownProps` and moved the `title` as optional prop @ b7425bd8196ff5942d4c257358cf401de6df8750 thus making it a general sorter drop down.

**Review by colinhowe — APPROVED** [review]

**Comment by colinhowe** [comment]
> > There are some headers missing. They'e created as part of [sparkletown/internal-sparkle-issues#1437](https://github.com/sparkletown/internal-sparkle-issues/issues/1437)
> 
> Different issue, different PR.
> 
> This PR is resolving
> 
> * [AdminV3 IA - Use new layout for spaces dashboard internal-sparkle-issues#1459](https://github.com/sparkletown/internal-sparkle-issues/issues/1459)

Apologies, I was leaving this comment for anyone else who looked at it and wondered what the plan for the headers was :)

**Pull request merged**